src/base/multithreading.cc

```diff
@@ -148,12 +147,15 @@ static std::string to_string(const MutexLock::ID i) {
 		return "Messages";
 	case MutexLock::ID::kIBaseVisualizations:
 		return "IBaseVisualizations";
+	case MutexLock::ID::kLog:
```

Comment:
Why do we need these new IDs?

Reply:
They're not new, they were added a while back. I just forgot to add them to the `to_string` function. The compiler didn't notice before that this `switch` was missing some `case`s because it was previously surrounded with a `#ifdef` that's normally disabled.